ts/Extensions/Annotations/Annotations.ts

```diff
@@ -165,6 +165,8 @@ declare global {
             point?: (string|MockPointOptions);
             itemType?: string;
             vertical?: VerticalAlignValue;
+            xAxis?: number;
```

Comment:
A few lines above- do we need this `extends` with an almost identical name?
When I merge that, everything works as expected, but maybe there is another reason for that?
``` TS
 interface AnnotationsLabelsOptions extends AnnotationsLabelOptions
```

Reply:
@bre1470 could you take a look?

Reply:
https://api.highcharts.com/highcharts/annotations.labelOptions is not identical to https://api.highcharts.com/highcharts/annotations.labels options.